.shopify-build/polaris-react.yml

```diff
@@ -0,0 +1,31 @@
+containers:
+  default:
+    docker: gcr.io/shopify-docker-images/apps/production/web-platform-ci-node:10.11.0-yarn-1.12.3
+  styleguide:
+    docker: circleci/node:10.13.0
+
+shared:
```

Comment:
You're not sharing anything between steps here so it seems odd to use the shared key. It might be simpler without faffing with yaml's referencing stuff, which means you can remove the shared key:


```yaml
steps:
  - label: 'Build web'
    container: default
    # etc
  - label: 'Build styleguide'
    container: styleguide
    # etc
```

Reply:
I prefer it this way since it clearly delineates what a step is doing, and where in the pipeline that step is executed. That, and the rest of our projects using Shopify Build follow the pattern I used, and I believe there's value in being consistent here.